backtest.js

```diff
@@ -6,8 +6,8 @@ const { Client } = require('pg')
 
 //////////////////////////////////////////////////////////////////////////////////
 
-const test_pair = "BTCUSDT"
```

Comment:
Why this change? Is it related to this PR?

Reply:
nope